### PR TITLE
integrating s3cmd malformed url test cases into cephci

### PR DESCRIPTION
**Pull request by hmaheswa**
pass logs: 

- pacific : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ytjph/
- quincy : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-kqfqj/

### CODE REVIEW

**Review by ckulal — COMMENTED** [review]
please include suite in 6.1 metafile aswell, otherwise we might miss in future

**Review by viduship — DISMISSED** [review]

**Review by anrao19 — APPROVED** [review]

File: pipeline/metadata/6.0.yaml [hunk 1]
```diff
@@ -440,6 +440,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "tier-2_rgw_test-s3cmd-malformed-url-with-different-commands"
+          execution_time: "2h 49m 32s"
+          suite: "suites/pacific/rgw/tier-2_rgw_test-s3cmd-malformed-url.yaml"
+          global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
+          platform: "rhel-8"
```
Comment (ckulal) [inline pipeline/metadata/6.0.yaml:447]: please change platform and inventory to rhel-9

File: pipeline/metadata/6.1.yaml [hunk 2]
```diff
@@ -416,6 +416,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "tier-2_rgw_test-s3cmd-malformed-url-with-different-commands"
+          execution_time: "2h 49m 32s"
+          suite: "suites/pacific/rgw/tier-2_rgw_test-s3cmd-malformed-url.yaml"
+          global-conf: "conf/pacific/rgw/tier-0_rgw.yaml"
+          platform: "rhel-8"
```
Comment (ckulal) [inline pipeline/metadata/6.1.yaml:423]: please change platform and inventory to rhel-9

File: pipeline/metadata/6.0.yaml (hunk 1 above)
Comment (hmaheswa) [reply pipeline/metadata/6.0.yaml]: thank you @ckulal , please review again

File: pipeline/metadata/6.1.yaml (hunk 2 above)
Comment (hmaheswa) [reply pipeline/metadata/6.1.yaml]: thank you @ckulal , please review again

**Review by ckulal — APPROVED** [review]

**Pull request merged**